commands/bench/lib/bench-all-cumulus.sh

```diff
@@ -74,7 +74,7 @@ run_cumulus_bench() {
 
 
 echo "[+] Compiling benchmarks..."
-cargo build --profile $profile --locked --features=runtime-benchmarks -p polkadot-parachain-bin
+cargo build --profile $profile --locked --features=runtime-benchmarks,riscv -p polkadot-parachain-bin
```

Comment:
I will add the feature to this node. We will need this once we add pallet_revive to westend anyways.